templates/common/render/render_test.go

```diff
@@ -1178,7 +1178,7 @@ steps:
 				Backups:             true,
 				BackupDir:           backupDir,
 				Clock:               clk,
-				DestDir:             dest,
+				OutDir:              dest,
```

Comment:
nits: Do we need to rename `dest`to `outputDir`

Reply:
Yeah, we should. Done.